features/steps/session_tracking_steps.rb

```diff
@@ -7,7 +7,6 @@
     And the payload field "device.manufacturer" equals "Apple"
     And the payload field "device.jailbroken" is not null
     And the payload field "device.modelNumber" is not null
-    And the payload field "device.wordSize" is not null
```

Comment:
Are we sure about removing `wordSize`?  I may not be quite on the money but it seems to be used server-side in determining the primary dysm for symbolication of crash reports?